app/views/admin/editions/_speed_tagging.html.erb

```diff
@@ -55,19 +55,6 @@
       </div>
     <% end %>
 
-    <% if @edition.can_be_associated_with_ministers? && (ministerial_roles = @edition.organisations.flat_map(&:ministerial_roles)).any? %>
```

Comment:
Now this line is gone, it looks like `Edition#can_be_associated_with_ministers?` method can now be removed as well.